modules/wechat_qrcode/include/opencv2/wechat_qrcode.hpp

```diff
@@ -0,0 +1,89 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Tencent is pleased to support the open source community by making WeChat QRCode available.
+// Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
+
+#ifndef __OPENCV_WECHAT_QRCODE_HPP__
+#define __OPENCV_WECHAT_QRCODE_HPP__
+#include "opencv2/core.hpp"
+#include "opencv2/wechat_qrcode/align.hpp"
```

Comment:
can we move all those headers: align, ssd_detector and super_scale inside `src`? It looks like they are not used for QR code detection and recognition. We can always expose more functionality later, but it's much more difficult to hide functionality that we do not need anymore, because some users may rely on it